rpc/server.go

```diff
@@ -0,0 +1,53 @@
+package rpc
+
+import (
+	"fmt"
+	"net"
+	"net/rpc"
+
+	"github.com/gluster/glusterd2/context"
+	"github.com/gluster/glusterd2/peer"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/kshlm/pbrpc/pbcodec"
+)
+
+func (r *Connection) PeerAddRPCSvc(args *peer.PeerAddRequest, reply *RPCPeerAddResp) error {
```

Comment:
Ahh, silly! I corrected it. Now that its hitting the server handler but still there is a deadlock, client is not get back the response :(